azure-pipelines-analyzer-release.yml

```
@@ -0,0 +1,153 @@
+trigger: none
+pr: none
+
+pool:
+  name: '1ES-Hosted-DurableTaskFramework'
+  demands:
+    - ImageOverride -equals MMS2022TLS
+
+steps:
+
+# Configure all the .NET SDK versions we need
+- task: UseDotNet@2
+  displayName: 'Use the .NET Core 2.1 SDK (required for build signing)'
+  inputs:
+    packageType: 'sdk'
+    version: '2.1.x'
+
+- task: UseDotNet@2
+  displayName: 'Use the .NET Core 3.1 SDK'
+  inputs:
+    packageType: 'sdk'
+    version: '3.1.x'
+
+- task: UseDotNet@2
+  displayName: 'Use the .NET 6 SDK'
+  inputs:
+    packageType: 'sdk'
+    version: '6.0.x'
+
+# Use NuGet
+- task: NuGetToolInstaller@1
+  displayName: 'Use NuGet '
+
+# dotnet restore
+- task: DotNetCoreCLI@2
+  displayName: 'dotnet restore'
+  inputs:
+    command: restore
+    projects: '**/**/*.csproj'
+    feedsToUse: config
+    nugetConfigPath: '.nuget/nuget.config'
+
+# Build durable-extension
+- task: VSBuild@1
+  displayName: 'Build Durable Extension'
```

We should probably update this display name.
```suggestion
  displayName: 'Build Durable Analyzer'
```